python/arcticdb/adapters/in_memory_library_adapter.py

```diff
@@ -0,0 +1,71 @@
+"""
+Copyright 2023 Man Group Operations Limited
+
+Use of this software is governed by the Business Source License 1.1 included in the file licenses/BSL.txt.
+
+As of the Change Date specified in that file, in accordance with the Business Source License, use of this software will be governed by the Apache License, version 2.0.
+"""
+import re
+from dataclasses import dataclass
+
+
+from arcticdb.options import LibraryOptions
+from arcticc.pb2.storage_pb2 import EnvironmentConfigsMap, LibraryConfig
+from arcticdb.version_store.helper import add_memory_library_to_env
+from arcticdb.config import _DEFAULT_ENV
+from arcticdb.version_store._store import NativeVersionStore
+from arcticdb.adapters.arctic_library_adapter import ArcticLibraryAdapter, set_library_options
+from arcticdb.encoding_version import EncodingVersion
+
+
+class InMemoryLibraryAdapter(ArcticLibraryAdapter):
+    """
+    Use in-memory storage.
+
+    Supports only the URI `"mem://"`.
```

Comment:
```suggestion
    Only supports the URI `"mem://"`.
```

is a bit more grammatical

Reply:
Agreed. I'll make the change in the docs PR.